querybook/server/datasources/metastore.py

```diff
@@ -214,9 +214,7 @@ def refresh_table_from_metastore(table_id):
 
         metastore_id = schema.metastore_id
         metastore_loader = get_metastore_loader(metastore_id, session=session)
-        metastore_loader.sync_create_or_update_table(
-            schema.name, table.name, session=session
-        )
+        metastore_loader.sync_table(schema.name, table.name, session=session)
```

Comment:
what happens if table is deleted, and we call `session.refresh(table)`?

also the frontend code assumes a table is always returned, can you add some checks on the frontend too? (like remove it from redux store, show messages, etc)